app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -114,12 +96,20 @@ export default class DiscoverySidebar extends React.Component {
     return moment(created_at).format("YYYY-MM-DD");
   }
 
-  static processSamples(newSamples, genomes) {
+  // TODO (gdingle): humanize better into M such as 34M
+  formatNumber(number) {
```

Comment:
Could you name this function something like `getAverageOverSamples`? To make it more clear what the function is doing.